### DISCUSSION TITLE
RequestContext.getCurrentInstance().update() after upgrade PF to 6.1

### DISCUSSION THREAD

**Discussion by Sergeyperekatov** [primefaces/community#2978] [PrimeFaces] [answered]
Hi, I'm asking for help. After upgrading PF to version 6.1 after 6.0, there was a problem using RequestContext.getCurrentInstance().update()
Early componentID view  
questionForm:j_idt278:0:j_idt406:resolutionFileNames

questionForm:j_idt278:0:candidatesTable:0:cmTable

questionForm:j_idt219:resolutionFileNames
worked without problems. 
Now when calling RequestContext.getCurrentInstance().update("questionForm:j_idt278:0:j_idt406:resolutionFileNames") I get a NullPointerException.
 RequestContext.getCurrentInstance().update ("questionForm:j_idt219: resolutionFileNames") continues to work

Part of the log
javax.el.ELException: /meeting/edit/question.xhtml @623,93 actionListener="#{questionView.materialAttach()}": java.lang.NullPointerException                                                                               at com.sun.faces.facelets.el.TagMethodExpression.invoke(TagMethodExpression.java:111)                                                                                                                              at javax.faces.event.MethodExpressionActionListener.processAction(MethodExpressionActionListener.java:147)

**Comment by tandraschko** [comment, marked as answer]
i just can suggest you to upgrade. There is no RequestContext since ages.
I think you need to debug by yourself.
